docker/gremlin-gaffer/README.md

```diff
@@ -0,0 +1,36 @@
+Gaffer
```

Comment:
Shouldn't this be Gremlin with Gaffer plugin?

Reply:
Yes sorry!